src/organization/screens/organization-profile.screen.js

```diff
@@ -82,22 +52,24 @@ class OrganizationProfile extends Component {
   }
 
   componentDidMount() {
-    const organization = this.props.navigation.state.params.organization;
+    const { org, orgId, getOrgById, getOrgMembers } = this.props;
 
-    this.props.fetchOrganizations(organization.login);
-    this.props.fetchOrganizationMembers(organization.login);
+    if (!org.name) {
+      getOrgById(orgId);
+    }
```

Comment:
This if does a little caching, to avoid refetching from the REST api if we already have the complete organization object (see comment below).

This is needed for now as single object retrieval is not cached in the new Redux API, and will be automatically handled soon.